src/Marian/index.ts

```diff
@@ -233,7 +234,7 @@ export default class Marian {
     res.end(responseBody);
   }
 
-  private async fetchResults(parsedUrl: URL): Promise<any[]> {
+  private async fetchResults(parsedUrl: URL): Promise<Document[]> {
```

Comment:
because the results returned are an aggregation, I don't think there's any way to be sure of the type except for the fact that it will be a `mongodb.Document` type.